package.json

```diff
@@ -16,7 +16,8 @@
     "test": "test"
   },
   "dependencies": {
-    "@openzeppelin/contracts": "^4.0.0",
+    "@openlaw/snapshot-js-erc712": "^1.6.0",
```

Comment:
The snapshot-js v1.6.0 may work to build the tribute-contracts v1 (neon branch), but if the UI is using `snapshot-js` > v1.2.0, it will probably break the Proposal flow - because v1.6.0 is probably targeting OffchainVoting from v2 contracts: https://github.com/openlawteam/tribute-contracts-cli#version-compatibility